### ISSUE TITLE
Omega is planned for removal for 8.14.

### ISSUE THREAD

**Issue by Zimmi48** [DeepSpec/sf#2]
Hello @liyishuai @bcpierce00 et al.,

Now that all the bugs that were preventing migration to `lia` have been solved, we'd like to proceed with the actual removal of `omega` in 8.14 (see coq/coq#13741). Most of the tested projects in Coq's CI have already been fixed (including ones that called `omega` a lot!) and no new issue has been discovered so far. Doing the change amounts to replacing `omega` with `lia` and `Omega` with either `ZArith` or `Lia`.

Obviously, we also need SF to adapt to this change. Would it be possible to take care of that soon? It would be great if we could avoid delaying this PR a lot more (it's already been opened for 3 months), especially as the cutoff date for the 8.14 release is approaching (it's in 2 months now).

Thanks!

**Comment by bcpierce00** [comment]
Yes, let's get this done.  

Volumes 1-2 and 4 of SF are omega-free now.  For volume 3 I am waiting for @andrew-appel to take a look at a couple of problems I encountered in changing omega to lia.  I don't know what the status of volume 5 is (Andrew?).

**Comment by Zimmi48** [comment]
Great to hear! Note that SF (and more specifically VFA) is now the last blocker before we can merge the PR removing `omega` so let us know as soon as this is done.

**Comment by andrew-appel** [comment]
Regarding VFA,  already 12 days ago Andrew Tolmach built a PR in SF that takes care of this, and I have asked @bcpierce00 to merge it.

**Comment by bcpierce00** [comment]
... which I will work on this morning... :-)

       - Benjamin

**Comment by bcpierce00** [comment]
Now merged.  I believe SF should be omega-free.

   - B

**Comment by liyishuai** [comment]
@bcpierce00 the build-and-push CI in sfdev was disabled (before I or someone fix it). We might need to update this public repo manually.

**Comment by bcpierce00** [comment]
Ah.  Can you do it or tell me what needs to be done?

**Comment by liyishuai** [comment]
1. Run `make prepare-export`; copy the tarballs to the corresponding `full` directory: https://github.com/DeepSpec/sfdev/blob/2409ce5e4caf3e7c3fe3ad23039e3fba303ce34a/.circleci/config.yml#L23-L69
2. Copy the entire `toplevel` directory to this `sf` repo, and push: https://github.com/DeepSpec/sfdev/blob/2409ce5e4caf3e7c3fe3ad23039e3fba303ce34a/.circleci/config.yml#L124-L137

**Comment by bcpierce00** [comment]
Do you have time to do this export process, Yishuai?  I'm not sure I follow
all the details here (e.g., where is the full directory supposed to be
created?)...

**Comment by liyishuai** [comment]
I'm having difficulty installing VST. I can try to skip VC volume and export the remainder.

**Comment by liyishuai** [comment]
Should be fixed by 3fb302e781ce6944cc3cd6fcf8bbb02774d7fbcc.

**Comment by bcpierce00** [comment]
Thanks!  Just to double-check: This means that the main Coq CI should be
working now without Omega?

**Comment by liyishuai** [comment]
Yes Coq CI should be working now.

**Comment by Zimmi48** [comment]
I can confirm this :tada: Thanks everyone for your help!

**Comment by bcpierce00** [comment]
Great!
